src/app/errors.rs

```diff
@@ -100,10 +100,14 @@ where
 
 fn map_err_to_5xx(e: Error) -> StatusCode {
     use proxy::http::router::error as router;
+    use proxy::shed::error as shed;
 
     if let Some(ref c) = e.downcast_ref::<router::NoCapacity>() {
         warn!("router at capacity ({})", c.0);
         http::StatusCode::SERVICE_UNAVAILABLE
+    } else if let Some(_) = e.downcast_ref::<shed::Overloaded>() {
+        warn!("server overloaded, max-in-flight reached");
```

Comment:
TIOLI: should we log the inflight limit here?

Reply:
We don't have the limit available here... 🤷‍♂️

Reply:
Ahh yeah, nevermind. Might eventually be nice to add that to the `shed::Overloaded` error type?